test/unit/on_config_spec.js

```diff
@@ -7,7 +7,7 @@ describe('UNIT: on_config', function () {
         $_rootScope,
         $_templateCache,
         $_location,
-        _state = "app"; //root component state
+        _state = "app"; //redirected root component state
```

Comment:
As far as I know, cognoma doesn't have a JS style guide yet. Feel free to create an issue and enforce one to later make the code easier to read (such as a space after `//`).